integration_tests/pages/plan.ts

```diff
@@ -1,4 +1,4 @@
-export default class PlanSummary {
+export default class Plan {
```

Comment:
Let's call this class PlanOverview and the file plan-overview.ts because we know plan-summary is coming